spec/skipped_test_worker.rb

```diff
@@ -0,0 +1,7 @@
+class SkippedTestWorker
+  @queue = :test_queue
+
+  def self.perform(first, second)
+    Thread.current["perform_skipped"] = true
```

Comment:
I'm a little confused about this test worker, since `perform` must be called in order to set `perform_skipped` to true. I assume in actual use we'd set `perform_skipped` from a hook and not from perform, though

Reply:
I should add a comment here, explaining this is just to set up the behavior to get the skipped event logged.

What makes it hard to test, is that the check for if it gets skipped is happening in an `around_perform` hook. So I'm trying to fake it instead of adding and removing a resque plugin for one spec.
https://github.com/ShippingEasy/ship-it/blob/staging/lib/resque_fail_if_lock_prevents_perform.rb